src/microbe_stage/OrganelleDefinition.cs

```diff
@@ -30,6 +30,16 @@ public class OrganelleDefinition : IRegistryType
     [TranslateFrom(nameof(untranslatedName))]
     public string Name = null!;
 
+    /// <summary>
+    ///   Description of processes the organelle performs
+    /// </summary>
+    public string? ProcessesDescription;
+
+    /// <summary>
+    ///   Description of the organelle
+    /// </summary>
+    public string Description = null!;
```

Comment:
The check method should be updated to ensure there must be a description for an organelle.

Reply:
I wanted to add that for Implemented organelles but myofibril and axon doesn't have process description

Reply:
I'm just asking for the description to be enforced, not the processes description. If some organelles just define the processes description then those should probably be defined to specify the description instead of the processes description. This is good to find these kinds of inconsistencies and fix them with a unified system.

Reply:
My bad